global_planner/src/planner_core.cpp

```diff
@@ -438,25 +441,40 @@ void GlobalPlanner::publishPotential(float* potential)
     potential_pub_.publish(grid);
 }
 
-void GlobalPlanner::publishInscribedRadius() const
+void GlobalPlanner::showFootprintRadii() const
 {
     double inscribed_radius, circumscribed_radius;
     costmap_2d::calculateMinAndMaxDistances(costmap_ros_->getRobotFootprint(), inscribed_radius, circumscribed_radius);
 
+    // show the inscribed radius (in blue)
     visualization_msgs::Marker marker;
```

Comment:
```suggestion
    visualization_msgs::MarkerArray markers;
```
F: use a marker array so you only publish once instead of twice

Reply:
I think it's much cleaner with marker;  otherwise to delete all I need to 
1. create marker array
2. create marker
3. add marker to array

instead of just create marker

Reply:
I don't think you need to manually clear as per my other suggestion.

But does the current code not cause problems with the id? You're publishing two markers with the same id in the same topic, so won't the second one override the first one? I'm not sure if namespace applies if they're different messages :thinking: 

Also, if you want to keep this, then we should increase the publisher queue size since right now is only 1 but we're publishing 2 messages in a row.

Reply:
the marker identification is namespace + id,,, so diff namespaces is enough 

I increased queue size